bounded-collections/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "bounded-collections"
-version = "0.2.3"
+version = "0.2.4"
 description = "Bounded types and their supporting traits"
 readme = "README.md"
 rust-version = "1.79.0"
```

Comment:
bump this to 1.81?

Reply:
I went through the workspace members and found the following rust versions to be the minimum:
```
members = [
	"fixed-hash", # 1.64
	"keccak-hash", # 1.79
	"kvdb", # 1.64
	"kvdb-memorydb", # 1.64
	"kvdb-rocksdb", # 1.71.1
	"kvdb-shared-tests", # 1.64
	"parity-bytes", # 1.64
	"rlp", # 1.64
	"rlp-derive", # 1.64
	"uint", # 1.64
	"primitive-types", # 1.79
	"bounded-collections", # 1.79
	"ethereum-types", # 1.79
	"ethbloom", # 1.64
]
```
1.64 is the minimum for the whole workspace since before this the cargo manifest requires author to be a different format.
weirdly for the individual packages I don't see the requirement for 1.81, so I wonder if there are some unused deps at the workspace level

Reply:
You can use [cargo msrv](https://github.com/foresterre/cargo-msrv) just to check that it works with that.  
We should also have this in CI actually.